client/jetpack-connect/signup.js

```diff
@@ -212,7 +260,111 @@ export class JetpackSignup extends Component {
 			</LoggedOutFormLinks>
 		);
 	}
+
+	renderWooDna() {
+		const { authQuery, isFullLoginFormVisible, translate, usernameOrEmail } = this.props;
+		const { isCreatingAccount, signUpUsernameOrEmail } = this.state;
+		let header, subHeader, content;
+		const footerLinks = [];
+		const email = signUpUsernameOrEmail || usernameOrEmail || authQuery.userEmail;
+		const wooDna = this.getWooDnaConfig();
+		let pageTitle;
+
+		if ( this.state.showWooDnaLoginForm ) {
```

Comment:
Here we're either showing the log in form or the sign up form. I think this could use a comment at the top of the if block to explain.